forum/qa-plugin/report-reason/script.js

```diff
@@ -1,42 +1,31 @@
-$(document).ready(function()
-{
-    $(".qa-form-light-button-flag").attr("type", "button");
+document.addEventListener("DOMContentLoaded", () => {
     
-    $(".qa-form-light-button-flag").click( function()
-    {
-        var postId = $(this).data("postid");
-        var postType = $(this).data("posttype");
-        var parentId = $(this).data("parentid");
+    const flagButton = document.getElementsByClassName('qa-form-light-button-flag')[0];
+    flagButton.type = "button";
```

Comment:
Jeśli nie tworzysz elementu DOM dynamicznie w JS, to  moim zdaniem czytelniej będzie dodać (lub zmienić) typ tego przycisku w HTML (czy tam w PHP, bo pewnie on generuje HTML).

Reply:
Z tego co pamiętam generowanie przycisków jest zrobione poprzez tablicę i nie można ustawić typu - podajemy tylko parametry takie jak nazwa, klasa czy title. Aczkolwiek popatrzę jeszcze